src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -278,3 +278,10 @@ export function isDocumentIntegration(
 ): integration is DocumentIntegration {
   return integration.hasOwnProperty('docUrl');
 }
+
+export const capitalizeString = (string: string) => {
+  return string
+    .split(' ')
+    .map(word => word.charAt(0).toUpperCase() + word.slice(1))
```

Comment:
@NisanthanNanthakumar you can use `lodash/capitalize`

Reply:
@scefali `lodash/capitalize` only capitalizes the first letter of the string. https://lodash.com/docs/4.17.15#capitalize

Reply:
@scefali we can use 'lodash/startcase` https://lodash.com/docs/4.17.15#startCase